packages/manager-components/vite.config.ts

```diff
@@ -6,6 +6,18 @@ import { UserConfigExport } from 'vite';
 
 const app = async (): Promise<UserConfigExport> => {
   return defineConfig({
+    resolve: {
+      dedupe: [
+        '@ovh-ux/manager-react-shell-client',
+        '@ovh-ux/manager-core-api',
+        'i18next',
+        'react',
+        'react-dom',
+        'react-i18next',
+        'react-router-dom',
+        'zustand',
+      ],
+    },
```

Comment:
i don't think this is needed

Reply:
👍  Yes, Still working without this, I delete this